.env

```diff
@@ -0,0 +1 @@
+PYTHONPATH=${PYTHONPATH}:${PWD}
```

Comment:
export missing

Reply:
I thought it was not needed here. Added.